swap/protocol_test.go

```diff
@@ -145,9 +145,13 @@ func TestEmitCheque(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-
+	// cashCheque cashes a cheque when we get twice the transaction costs.
+	// gasPrice on testBackend == 1
+	// estimated gas costs == 50000
+	// cheque should be send if the accumulated uncashed cheques is worth more than 100000
```

Comment:
cheque should be sent (typo)

Reply:
`cheques should be sent updated`
`are worth` not updated, as the `accumulated amount is worth` and not `accumulated amount are worth`. Added `amount`, to make this clear.